operate/services/deployment_runner.py

```diff
@@ -249,6 +251,9 @@ def _start_tendermint(self) -> None:
         """Start tendermint process."""
         working_dir = self._work_directory
         env = json.loads((working_dir / "tendermint.json").read_text(encoding="utf-8"))
+        env["PYTHONUTF8"] = "1"
+        env["PYTHONIOENCODING"] = "utf8"
```

Comment:
Were you able to verify that this actually gets picked up?

Reply:
it's experimental one, according to python docs

Reply:
seems unconventional but i'm no python expert..

why not use `os.environ["PYTHONIOENCODING"] = "1"` in the projects we're pulling from? in an entry script for example?

fine to approve, just curious!

Reply:
as i read, it's better to set it up on python start, so not sure it will be properly applied within python code, so decided to use env vars according to the python docs